src/app/basic_examples/abi_decode/page.mdx

```diff
@@ -0,0 +1,141 @@
+export const metadata = {
+  title: "ABI Encode • Stylus by Example",
+  description:
+    "A simple solidity ABI encode and decode example",
+};
+
+{/* Begin Content */}
+
+# ABI Decode
+
+The `decode` can not be used for `encode_packed` data because it ignores padding when encode.
```

Comment:
Can you add some introduction for context? You can duplicate the same first paragraph as in the "encode" page, or reference that page for initial context.